.github/dockerimg/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # syntax=docker/dockerfile:1.4
-FROM --platform=$BUILDPLATFORM ubuntu:22.04 AS build
+FROM --platform=$TARGETPLATFORM ubuntu:22.04 AS build
```

Comment:
This should be the build platform, as that's where we're building the code I'd though